src/test/scala/nl.knaw.dans.easy.bag2deposit/ddm/ProvenanceV2Spec.scala

```diff
@@ -0,0 +1,97 @@
+
+class ProvenanceV2Spec extends AnyFlatSpec with FileSystemSupport with XmlSupport with Matchers with FixedCurrentDateTimeSupport with DebugEnhancedLogging with SchemaSupport with AppConfigSupport {
+  override val schema: String = "target/dans-schema-lib/bag/metadata/prov/v2/provenance.xsd"
+
+  // FixedCurrentDateTimeSupport is not effective for a val
+  private def provenanceBuilder = Provenance("EasyConvertBagToDepositApp", "1.0.5")
+
+  "Provenance" should "change namespace" in {
+    val ddmIn = {
+      <ddm:DDM xmlns:ddm="http://easy.dans.knaw.nl/schemas/md/ddm/"
+               xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+               xsi:schemaLocation=" http://easy.dans.knaw.nl/schemas/md/ddm/ https://easy.dans.knaw.nl/schemas/md/ddm/ddm.xsd"
+               xmlns:gml="http://www.opengis.net/gml"
+      >
+        <ddm:profile>
+          <dc:title>blabla</dc:title>
+        </ddm:profile>
+      </ddm:DDM>
+    }
+
+    val expectedDdm = <ddm:DDM xmlns:ddm={ ddmV2namespace }
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+             xmlns:gml="http://www.opengis.net/gml"
+    >
+      <ddm:profile>
+        <dc:title>blabla</dc:title>
+        <ddm:personalData present="Yes"/>
+      </ddm:profile>
+    </ddm:DDM>
+
+    val expectedProv = {
+      <prov:provenance xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                       xmlns:prov="http://schemas.dans.knaw.nl/bag/metadata/prov/v2/"
+      >
+        <prov:migration app="EasyConvertBagToDepositApp" version="1.0.5" date="2020-02-02">
+        <prov:file scheme={ ddmV2namespace } xmlns=""
+                   xmlns:ddm={ ddmV2namespace }
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns:gml="http://www.opengis.net/gml"
+        >
+        <prov:old/>
+        <prov:new><ddm:personalData present="Yes"/></prov:new>
+          </prov:file>
+        </prov:migration>
+      </prov:provenance>
+    }
+
+    // a few steps of EasyConvertBagToDepositApp.addProps
+    val ddmOut = testConfig("archaeology").copy(ddmVersion = V2).ddmTransformer
+      .transform(ddmIn, "easy-dataset:123", containsPrivacySensitiveData = "true")
+      .getOrElse(fail("no DDM returned"))
+    val actualProv = provenanceBuilder.collectChangesInXmls(List(Provenance.compareDDM(ddmIn, ddmOut)))
+
+    // check what was stripped by normalize to avoid random order
+    ddmOut.serialize should include(ddmV2namespace)
+    actualProv.serialize should include(ddmV2namespace)
+
+    normalized(ddmOut) shouldBe normalized(expectedDdm)
+    normalized(actualProv).replaceAll(" +scheme"," scheme") shouldBe normalized(expectedProv).replaceAll(" +scheme"," scheme")
+
+    SchemaFactory
+      .newInstance(XMLConstants.W3C_XML_SCHEMA_NS_URI)
+      .newSchema(Array[Source](new StreamSource(new File(schema))))
+      .newValidator()
+      .validate(new StreamSource(new ByteArrayInputStream(actualProv.serialize.getBytes())))
```

Comment:
Schema loads after replacing FileInputStream by File. but validation now gives

    cvc-elt.1.a: Cannot find the declaration of element 'prov:provenance'.
     org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 134; cvc-elt.1.a: Cannot find the declaration of element 'prov:provenance'.